### PR TITLE
Defining an incident response

### LINKED ISSUE #1382: Typeset and publish page Defining an AI Incident Response
From #1282 "We need a page on the website linked from the footer that defines what an incident response is."

Page and link name is **Defining an "AI Incident Response"**
Page should be linked in the footer right under **Defining an "AI Incident"**

See Google Doc for content here:
https://docs.google.com/document/d/1vPk5CrZKGGYB5r55WvTNr5M0jsMJP4e7TDqswmt-_88/edit?usp=sharing

There is a comment in the google doc to add visuals. We would like a screenshot of the visual indicator of an incident response that the engineering team produces (this is one of the requests in the original GitHub issue). However, do not let this stop this issue from being published if it is not completed yet.

Let @LKchemposer or I know if you have any questions about the content.

### PR DESCRIPTION
**Pull request by lmcnulty**
Closes #1382.

- Visuals for Incident Responses don't exist yet
- I've set the slug to `/research/5-response`. I'm wondering if it would be better to move the research pages to the top-level (redirecting the previous locations) so that the pages will be easier to re-order and rearrange as the site content develops.

### CODE REVIEW

File: site/gatsby-site/src/components/ui/MdxComponents.js [hunk 1]
```diff
@@ -3,8 +3,16 @@ import isString from 'lodash/isString';
 import Link from './Link';
 import SensitiveImage from '../images/sensitive';
 
-const slug = (title) =>
-  isString(title) ? title.toLowerCase().replace(/\s+/g, '') : title.props.children;
+const slug = (title) => {
+  console.log(`title`, title);
```
Comment (pdcp1) [inline site/gatsby-site/src/components/ui/MdxComponents.js:7]: Should remove this `console.log()`

File: site/gatsby-site/src/components/ui/MdxComponents.js [hunk 2]
```diff
@@ -3,8 +3,16 @@ import isString from 'lodash/isString';
...
+  if (isString(title)) {
+    return title.toLowerCase().replace(/\s+/g, '');
+  }
+  if (Array.isArray(title)) {
+    return title.map((e) => slug(e)).join('');
+  }
```
Comment (pdcp1) [inline site/gatsby-site/src/components/ui/MdxComponents.js:13]: In which case do we have this title as an Array?

**Review by pdcp1 — CHANGES_REQUESTED** [review]

File: site/gatsby-site/src/components/ui/MdxComponents.js (hunk 2 above)
Comment (lmcnulty) [reply site/gatsby-site/src/components/ui/MdxComponents.js:13]: When a heading contains a link, the text and the link become array elements. E.g.

```markdown
### Report 1: [Knightscope Issues Field Incident Report](/cite/51#r1765)
```

Comment (pdcp1) [reply site/gatsby-site/src/components/ui/MdxComponents.js:12]: Got it!

**Comment by pdcp1** [comment]
How are we handling translations on Blog posts? @clari182 do the FR version and does any other Spanish native speaker do the ES version?

**Comment by clari182** [comment]
@pdcp1 I'll work on the FR version and create a PR against this one so Luna can include it on her branch

**Comment by clari182** [comment]
Hey @lmcnulty the french version is ready here: https://github.com/lmcnulty/aiid/pull/15/
I just found that there was a `#todo` link on line 22. On the french version, I set that to `/app/submit`

**Comment by pdcp1** [comment]
@lmcnulty the Spanish version is also ready here: https://github.com/lmcnulty/aiid/pull/16
Feel free to merge it to this PR.

File: site/gatsby-site/content/research/5-response/index.mdx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+2. Why the incident happened,
+3. What the entity has done or is going to do to
+   <ol style={{ marginLeft: '2em' }} type="a">
+      <li>remediate the incident, and</li>
+      <li>prevent future incidents from recurring.</li>
+   </ol>
+
+## Submitting Incident Responses
+
+The AI Incident Database is designed to provide information, structure, and perspectives on AI incidents. If you are listed as an entity for one or more incidents, we would very much like to work with you to provide your perspective. You can either [submit a response directly](#todo) to the database, or [contact us](/contact) to discuss submitting a response.
```
Comment (pdcp1) [inline site/gatsby-site/content/research/5-response/index.mdx:22]: We should change `#todo` link to `/app/submit`

**Review by pdcp1 — CHANGES_REQUESTED** [review]

File: site/gatsby-site/content/research/5-response/index.mdx (hunk 3 above)
Comment (lmcnulty) [reply site/gatsby-site/content/research/5-response/index.mdx]: Thanks, fixed.

**Review by pdcp1 — APPROVED** [review]

**Pull request merged**